CHANGELOG.md

```diff
@@ -11,6 +11,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Security
+
+## [0.1.40] - 2024-07-15
+
+### Added
```

Comment:
Can you please remove the empty sections in this? Trying to standardize the CHANGELOG format.

Note that `hatch run scripts:update-version` will do this automatically for you :D